### PR TITLE
Feat: update cm details

### LINKED ISSUE #102: [Task]: Update candy machine configuration to new design
**Task description.**

image

**Requirements.**
- Display `loading...` while loading
- Display and fill form
- Update works
- Store information in recoil

### PR DESCRIPTION
**Pull request by evavirseda**
Fixes #102 

## Changes

  - Update candy machine configuration to new design
  - Fix buttons size
  
### Checks

* [x] Have you catchup your branch with the latest state in base?
* [x] Have you lint your code locally prior to submission?
* [x] Have you reviewed your proposed changes and removed debris?
* [x] Have you successfully ran due tests with your changes?

### CODE REVIEW

File: components/forms/UpdateCandyMachine.tsx [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    candyMachinePubkey?: string | string[]
+}> = ({ fetchedValues, candyMachinePubkey }) => {
+    const { publicKey } = useWallet()
+    const anchorWallet = useAnchorWallet()
+    const { rpcEndpoint } = useRPC()
+
+    const { cache, uploadCache } = useUploadCache()
+    const [isInteractingWithCM, setIsInteractingWithCM] = useState(false)
+    const [status, setStatus] = useState('')
+    const [errorMessage, setErrorMessage] = useState('')
```
Comment (cpl121) [inline components/forms/UpdateCandyMachine.tsx:25]: `errorMessage` state is not being used

**Review by cpl121 — CHANGES_REQUESTED** [review]
Nice job!! just 2 comments

**Comment by VmMad** [comment]
When you cancel the transaction you still get the message `Candy Machine updated successfully!`

**Comment by agus-xyz** [comment]
looks good to me @evavirseda please lets fix the conflicts with base before proceeding to merge

**Comment by agus-xyz** [comment]
@evavirseda it seems like the cm info in devnet broke after adding the selectable network. Do you think you can fix this yourself, or shall we ping @aspnxdd? 
image

**Comment by evavirseda** [comment]
> @evavirseda it seems like the cm info in devnet broke after adding the selectable network. Do you think you can fix this yourself, or shall we ping @aspnxdd? image

yes! I have already spoken with @aspnxdd , it seems that the first requests are for the mainnet RPC. He is working on it

**Pull request merged**